app/js/controller/list_controller.js

```diff
@@ -165,20 +165,27 @@ export default class ListController extends Controller {
     }
 
     installReq.onerror = (err) => {
+      var errorMsg = err.target.error.name;
       var errorType = err.target.error.name;
```

Comment:
You're right, this is kinda confusing. The point is that `err.target.error.name` is the default error message, but that's not clear here. I'll restructure.